src/main/scala/beam/utils/map/R5MapStatsCalculator.scala

```diff
@@ -16,8 +16,10 @@ object R5MapStatsCalculator {
     // LoggerFactory.getLogger("com.conveyal").asInstanceOf[Logger].setLevel(Level.INFO)
     val pathToOsm = args(0)
     val minimumTagFrequencyToPrintTagOut = 1000
+    // TODO use `beamConfig.beam.routing.r5.linkRadiusMeters`
```

Comment:
Why not set it as part of this PR? Or was this a note for you to remember before closing the PR? Either way, this seems like it should be addressed.

Reply:
The comment was meant for someone who try to use this class and run across radius related errors. I should have removed the `TODO` from in front of it. It seems original author is @nikolayilyin, and he can chime in to suggest how we want to do this. 

I believe we don't want to pass beam config to the program just to get linkRadiusMeters value. 

We can do one of the followings,
1. add `args(1)` to pass linkRadiusMeters
2. elaborate comment for end user for existing change.

Thoughts?

Reply:
I would say we should have it take args(1), but with a default of 10000.